core/graph/src/test/java/uk/gov/gchq/gaffer/operation/export/resultcache/handler/ExportToGafferResultCacheHandlerTest.java

```diff
@@ -132,7 +132,7 @@ public void shouldHandleOperationByDelegatingToAnNewExporter() throws OperationE
         // Then
         assertSame(handlerResult, results);
         final ArgumentCaptor<OperationChain> opChain = ArgumentCaptor.forClass(OperationChain.class);
-        verify(cacheStore, times(3)).execute(opChain.capture(), Mockito.any(Context.class));
+        verify(cacheStore, atLeast(1)).execute(opChain.capture(), Mockito.any(Context.class));
```

Comment:
What's the significance of this change?

Reply:
This was something I couldn't quite get to the bottom of, originally this test was implemented to just check the `execute()` method was called at least once. Then under [this commit](https://github.com/gchq/Gaffer/commit/dd18afb4367c1700da21ebc9ca69f59c9c14967a) it was changed to exactly 3 with no explanation, with the current changes it now seems to run 4 times so changed to original implementation.

Reply:
Interesting, @t92549 - any ideas?

Reply:
In that old PR I probably just made the test case more specific/precise, however, I am pretty sure there is no reason it has to be exactly 3, we just want to make sure it was called once, so I think this change is good.